python/tvm/relay/frontend/tflite.py

```diff
@@ -258,15 +260,18 @@ def get_tensors(self, tensors_idx_list):
                     assert isinstance(tflite_zero_point, np.ndarray)
 
                     # Tensor - Per-axis quantization
-                    if tflite_scale.shape != (1,) and tflite_zero_point.shape != (1,):
+                    if tflite_scale.size != 1 and tflite_zero_point.size != 1:
                         scale = tflite_scale
-                        # Ensure that all zero points are identical
+                        # Ensure that all zero points are zeros
                         zero_point = tflite_zero_point
-                        assert all(x == zero_point[0] for x in zero_point)
+                        if not all(x == 0 for x in zero_point):
```

Comment:
Suggestion `if not all(x == 0 for x in zero_point):` >> `if not np.all((zero_point == 0)):`